cypress/integration/examples/spies_stubs_clocks.spec.js

```diff
@@ -0,0 +1,205 @@
+/// <reference types="cypress" />
+// remove no check once Cypress.sinon is typed
+// https://github.com/cypress-io/cypress/issues/6720
+
+context('Spies, Stubs, and Clock', () => {
```

Comment:
The test suite is not related to the task. It's for testing spies, stubs, and clocks, not the Book Store app functionality.